src/api/app/models/comment.rb

```diff
@@ -5,6 +5,9 @@ class Comment < ApplicationRecord
   belongs_to :user, inverse_of: :comments
   belongs_to :moderator, class_name: 'User', optional: true
 
+  # Preloads for polymorphic commentable:
```

Comment:
I think the comment is unnecessary, as the `includes` will fail on L36 if this is missing.  But I don't feel strongly about it.

Reply:
I'll remove it then. :+1: